src/Native/Unix/System.Native/pal_termios.cpp

```diff
@@ -0,0 +1,360 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
We're now trying to use C rather than C++ moving forward, so that more and more can be shared with Mono.  It'd be good to not add to that debt.  See https://github.com/dotnet/corefx/pull/25032 for more background.
cc: @luhenry